packages/app-builder-lib/src/macPackager.ts

```diff
@@ -91,6 +92,42 @@ export default class MacPackager extends PlatformPackager<MacConfiguration> {
     }
   }
 
+  protected async doPack(outDir: string, appOutDir: string, platformName: ElectronPlatformName, arch: Arch, platformSpecificBuildOptions: MacConfiguration, targets: Array<Target>): Promise<any> {
+    switch (arch) {
+      default: {
+        return super.doPack(outDir, appOutDir, this.platform.nodeName as ElectronPlatformName, arch, this.platformSpecificBuildOptions, targets);
```

Comment:
Hi @lutzroeder, 

Thank you for the universal support.

Why do you overwrite `platformName` and `platformSpecificBuildOptions` instead of passing arguments directly? It breaks mas builds.

Reply:
Good catch, these should pass through. Can you create an issue to track and/or feel free to send a PR.